docs/markdown/Getting Help/the-pants-community.md

```diff
@@ -47,7 +47,11 @@ Pants has a core group of [_Maintainers_](doc:maintainers): trusted people with
 
 Anyone who has made a few contributions of any kind, and has an ongoing interest in the Pants project, may be nominated by a Maintainer to become a _Contributor_. 
 
-Contributors may be granted extra permissions, such as the ability to assign issues, but these stop short of full Maintainer permissions. Contributors do not have any new obligations—for example, you are _not_ expected to make a certain number of changes. However, contributors are expected to respect their extra permissions. Contributors will be eligible to receive more attention and mentorship in activities like code review. Contributors with a continuing track record of contribution may be nominated to become Maintainers.
+Contributors may be granted extra permissions, such as the ability to assign issues and to fully participate in monthly contributor meetings. These permissions stop short of full Maintainer permissions, such as not yet getting write access to the repository.
```

Comment:
@cczona clarified to me over DM just now that team meetings are intended to be only for maintainers and contributors: give a protected space. So, probably this would be clearer?

```suggestion
Contributors may be granted extra permissions, such as the ability to assign issues and to participate in monthly contributor meetings. These permissions stop short of full Maintainer permissions, such as not yet getting write access to the repository.
```

Reply:
I think this implies that a Contributor may or may not be welcome at the gathering. It would be more accurate to express that all current maintainers and all current contributors are invited. In other words, it's an automatic privilege of membership on the team rather than a potential priv. Also, to clarify: we don't have any contributor meetings. We hold a monthly team meetup.

Reply:
Feel free to also work in something like "See the 'Meet The Team' page for a list of current and past team members" after mentioning that the meetups are for current team members. That way everyone is on the same page about who has this priv.